server/controllers/auth/strategies/facebookStrategy.js

```diff
@@ -0,0 +1,91 @@
+import passport from 'passport';
+import { Strategy as FacebookStrategy } from 'passport-facebook';
+import { config } from 'dotenv';
+
+import db from '../../../models';
+import hashPassword from '../../../helpers/paswordHash';
+
+config();
+
+const { FACEBOOK_APP_ID, FACEBOOK_APP_SECRET } = process.env;
+const { User, Profile } = db;
+
+/**
+ *
+ *
+ * @class Facebook
+ */
+class Facebook {
+  /**
+   * @returns {fn} - Middleware for facebook strategy
+   */
+  static facebookStrategy() {
+    passport.use(new FacebookStrategy({
+      clientID: FACEBOOK_APP_ID,
```

Comment:
Hi @theghostyced  Awesome job so far, however, i think you should specify on the PR that, the ``` FACEBOOK_APP_ID,```  and  ``` FACEBOOK_APP_SECRET ``` should be setup as an environment  variable so that the person testing knows. I'm also seeing this error on Travis build
<img width="889" alt="eze2" src="https://user-images.githubusercontent.com/21243814/49622421-db6e9400-f9c9-11e8-8794-2bf5391af3b6.png">

Reply:
Nice input @ekpangmichael. Didn't put that into consideration